### PR TITLE
Branchbranch

### PR DESCRIPTION
**Pull request by meaghanlewis**

### CODE REVIEW

File: TestWebAppProject/Program.cs [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
+﻿using Microsoft.AspNetCore.Hosting;
 using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
 
 namespace TestWebAppProject
 {
-    public class Program
+	public class Program
```
Comment (meaghanlewis) [inline TestWebAppProject/Program.cs]: fix the indentation

**Review by meaghanlewis — COMMENTED** [review]
just leaving a comment

File: TestWebAppProject/Startup.cs [hunk 2]
```diff
@@ -24,8 +24,9 @@ public Startup(IHostingEnvironment env)
 
         public IConfigurationRoot Configuration { get; }
 
-        // This method gets called by tffhe runtime. Use this method to add services to the container.
-        public void ConfigureServices(IServiceCollection services)
+		// This method gets called by tffhe runtime. Use this method to add services to the container.
```
Comment (ghfvs-tester) [inline TestWebAppProject/Startup.cs:27]: new review

File: TestWebAppProject/Startup.cs [hunk 3]
```diff
@@ -24,8 +24,9 @@ public Startup(IHostingEnvironment env)
...
+		// This method gets called by tffhe runtime. Use this method to add services to the container.
+		public void ConfigureServices(IServiceCollection services)
```
Comment (ghfvs-tester) [inline TestWebAppProject/Startup.cs:29]: 2nd comment! <p> paragraph </p>

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]
comments and request

File: TestWebAppProject/Program.cs [hunk 4]
```diff
@@ -1,25 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using Microsoft.AspNetCore.Hosting;
 using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Hosting;
 
 namespace TestWebAppProject
```
Comment (ghfvs-tester) [inline TestWebAppProject/Program.cs:4]: ???

File: TestWebAppProject/Program.cs [hunk 5]
```diff
@@ -1,25 +1,7 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using Microsoft.AspNetCore.Hosting;
```
Comment (ghfvs-tester) [inline TestWebAppProject/Program.cs:1]: ???? lalala

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]

File: TestWebAppProject/Class.cs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TestWebAppProject
```
Comment (ghfvs-tester) [inline TestWebAppProject/Class.cs:6]: cool!!!

File: TestWebAppProject/Class.cs [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+{
+    public class Class
```
Comment (ghfvs-tester) [inline TestWebAppProject/Class.cs:8]: public?

Comment (ghfvs-tester) [reply TestWebAppProject/Class.cs:8]: another review

**Review by ghfvs-tester — APPROVED** [review]

**Pull request closed without merging**